src/components/QuestionPlay/index.tsx

```diff
@@ -1,135 +1,156 @@
-        isAnswerRevealed && (
-          <div className="flex">
-            <Button type="button" size="large" className="fullWidth" color="red" onClick={() => submitResponse(null)}>
-              Incorrect
-            </Button>
+              </Box>
+              {selectedOptionId === '' && (
+                <Group>
+                  <Button radius="md" color="red" onClick={() => handleSelectedChoice('')}>
+                    Incorrect
+                  </Button>
+                  <Button radius="md" color="green" onClick={() => handleSelectedChoice(options[0].optionId)}>
+                    Correct
+                  </Button>
+                </Group>
+              )}
+            </>
+          ) : (
             <Button
-              type="button"
-              size="large"
-              className="ml-lg fullWidth"
               color="green"
-              onClick={() => submitResponse(options[0].optionId)}>
-              Correct
+              radius="md"
+              onClick={() => {
+                setIsAnswerRevealed(true);
+
+                // pause timer when the answer is revealed
+                if (pauseTimer) {
+                  pauseTimer();
+                }
+              }}>
+              Reveal answer
             </Button>
-          </div>
-        )
-      ) : (
-        // for preview mode show save button & while playing show submit button for mcq
-        <Button type="submit" size="large" color="green">
-          {isPreview ? 'Save' : 'Submit'}
-        </Button>
-      )}
-    </form>
+          )
+        ) : (
+          <>
+            <Checkbox.Group
+              value={selectedChoice === null ? undefined : [selectedChoice.toString()]}
+              orientation="vertical"
+              label={<Title order={6}>Options</Title>}
+              size="md"
+              my="lg"
+              onChange={(value: string[]) => setSelectedChoice(value[value.length - 1])}>
+              {options.map((option) => (
+                <Checkbox
+                  key={option.optionId}
+                  label={<Markdown>{option.text}</Markdown>}
+                  className={classNames({
+                    [styles.isAttempted]: isAttempted,
+                    [styles.correct]: option.isCorrect && selectedOptionId === option.optionId,
+                    [styles.inCorrect]: !option.isCorrect && selectedOptionId === option.optionId,
+                  })}
+                  value={option.optionId.toString()}
+                  disabled={isAttempted}
+                  radius="xl"
+                  color="red"
+                  size="lg"
+                />
+              ))}
+            </Checkbox.Group>
+            {!isAttempted && (
+              <Button radius="md" color="green" onClick={submitQuestionResponse}>
+                Submit
+              </Button>
+            )}
+          </>
+        )}
+        <button className="displayNone" id="btnSubmitResponse" type="submit">
```

Comment:
Why is there 2 submit buttons?

Reply:
this it  submit button for form, which is hidden.
when user click on Correct / InCorrect (when Q is without option) or Submit button (when Q is with option). I'm triggering click on this button